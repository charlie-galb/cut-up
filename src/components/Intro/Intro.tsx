import React from 'react'

export const Intro = () => {
    return (
        <div className='intro-container'>
            <p><b>Cut Up Creator is a tool for making collage poems in the style 
                of William S. Burroughs. </b></p>
            <br />
            <p>To begin, just copy text from three different sources (the more 
                different the better) and paste them in the 
                boxes below. Then hit the cut-up button to shred the text.
            </p>
        </div>
    )
}
