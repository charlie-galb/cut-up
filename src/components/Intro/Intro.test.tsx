import React from 'react';
import { render } from "@testing-library/react";

import { Intro } from './Intro';

test('renders without crashing', () => {
    const intro = render(<Intro />);
    expect(intro).toMatchSnapshot()
});