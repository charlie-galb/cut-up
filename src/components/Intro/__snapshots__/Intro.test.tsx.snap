// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="intro-container"
      >
        <p>
          <b>
            Cut Up Creator is a tool for making collage poems in the style of William S. Burroughs. 
          </b>
        </p>
        <br />
        <p>
          To begin, just copy text from three different sources (the more different the better) and paste them in the boxes below. Then hit the cut-up button to shred the text.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="intro-container"
    >
      <p>
        <b>
          Cut Up Creator is a tool for making collage poems in the style of William S. Burroughs. 
        </b>
      </p>
      <br />
      <p>
        To begin, just copy text from three different sources (the more different the better) and paste them in the boxes below. Then hit the cut-up button to shred the text.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
